tests/README.md

```diff
@@ -0,0 +1,54 @@
+# vault-manager testing
+
+This project uses BATS (Bash Automated Testing System) for integration testing.
+
+Upon commit, a build is triggered in the CI pipeline which runs the tests.
+The build sets up necessary resources to run the tests, then executes `pr_check.sh`.
+
+## Bats testing
+
+https://bats-core.readthedocs.io/en/stable/docker-usage.html
+
+## Building images and running tests locally
+
+From the top level directory of the project, run:
+
+```bash
+make test
```

Comment:
this doesn't exist anymore... This README seems to not have evolved with the above code (although it's great to have it)

Reply:
Working on the docs as changes are made and are in working order.  Planning for a complete overhaul of the docs before merging.